features/dd-sdk-android-trace-otel/src/main/kotlin/com/datadog/android/trace/opentelemetry/OtelTracerProvider.kt

```diff
@@ -217,7 +191,7 @@ class OtelTracerProvider internal constructor(
          * @param threshold the threshold value (default = 5)
          */
         fun setPartialFlushThreshold(threshold: Int): Builder {
-            builderDelegate.withPartialFlushThreshold(threshold)
+            builderDelegate.withPartialFlushMinSpans(threshold)
```

Comment:
I realized that `withPartialFlushThreshold` and `withPartialFlushMinSpans` actually makes the same (otel and opentracing notation)